starknet_py/tests/e2e/docs/guide/test_serializing.py

```diff
@@ -1,4 +1,4 @@
-# pylint: disable=import-outside-toplevel, pointless-string-statement
+# pylint: disable=import-outside-toplevel, pointless-string-statement, unbalanced-tuple-unpacking
```

Comment:
It was added because of new type adjustments, [this line](https://github.com/software-mansion/starknet.py/blob/3afdbf2df26bd203c4ed64bf4157b874542368bb/starknet_py/tests/e2e/docs/guide/test_serializing.py#L54) generates error, but we actually know how this tuple looks like. That's why I disabled lint